pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/ConsumerHandler.java

```diff
@@ -186,7 +186,9 @@ public void onWebSocketText(String message) {
     public void close() throws IOException {
         if (consumer != null) {
             this.service.removeConsumer(this);
-            consumer.close();
+            consumer.closeAsync().thenAccept(x ->
+                log.debug("[{}] Closed consumer asynchronously", consumer.getTopic())
```

Comment:
can we add debug statement if debug is enabled
`if (log.isDebugEnabled()) {..}`